openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.tsx

```diff
@@ -183,15 +183,20 @@ const ActivityFeedList: FC<ActivityFeedListProp> = ({
               tag="button"
               variant="link"
               onClick={() => setShowThreadTypeList((visible) => !visible)}>
-              <SVGIcons alt="filter" icon={Icons.FILTERS} width="14px" />
-              <span className="tw-font-medium tw-text-grey tw-ml-1">
+              <SVGIcons
+                alt="filter"
+                className="tw-text-primary"
+                icon={Icons.FILTER_PRIMARY}
+                width="14px"
```

Comment:
I think we don't have any width param in SVGIcons can you double check?

Reply:
@ShaileshParmar11  we are accepting it.